pages/project-details/[id]/index.js

```diff
@@ -47,43 +45,25 @@ export default function ProjectDetails({ views, projects, currentProject }) {
             projects={projects}
             currentProject={currentProject}
           />
-        </section>
+        </ViewsSection>
       </Main>
     </>
   );
 }
 
-const StyledImage = styled(Image)`
-  object-fit: cover;
-`;
-
 const DescriptionText = styled.p`
-  overflow: scroll;
+  overflow: hidden;
```

Comment:
The scroll one, was ok.
What I meant, was why not using `overflow-y` instead.
I mean you don't need to scroll left or right.